keyboards/monsgeek/m5/info.json

```
@@ -0,0 +1,291 @@
+{
+    "keyboard_name": "M5",
+    "manufacturer": "MonsGeek",
+    "maintainer": "jonylee@hfd",
+    "usb": {
+        "vid": "0xFFFE",
+        "pid": "0x000A",
+        "device_version": "1.0.4",
+        "suspend_wakeup_delay": 400,
+        "force_nkro": true
+    },
+    "processor": "WB32FQ95",
+    "bootloader": "wb32-dfu",
+    "features": {
+        "bootmagic": true,
+        "mousekey": false,
```

Generally preferred to have mousekeys enabled, unless there's a good reason not to (eg. USB endpoints)
```suggestion
        "mousekey": true,
```